resty

```diff
@@ -168,6 +170,17 @@ for my $var (sort keys %ENV) {
     $env_list .= "env $var;\n";
 }
 
+my $main_include_directive = '';
+my $http_include_directive = '';
+
+if ($main_include) {
+    $main_include_directive = "include $main_include;"
```

Comment:
Maybe it's better to test the existence of the file path and also turn relative paths into absolute ones via the standard Perl module `File::Spec`?